src/components/Editor/SearchBar.js

```diff
@@ -170,6 +196,7 @@ const SearchBar = React.createClass({
 
     if (e) {
       e.preventDefault();
+      e.stopPropagation();
```

Comment:
The command opens the devtools preferences panel when you're in the Firefox panel.